modules/reference/pages/properties/topic-properties.adoc

```diff
@@ -547,6 +547,23 @@ NOTE: Although `replication.factor` isn't returned or displayed by xref:referenc
 
 ---
 
+[[deleteretentionms]]
+==== delete.retention.ms
+
+The retention time (in ms) for tombstone records in a compacted topic. Redpanda removes tombstone records after the retention limit is exceeded.
```

Comment:
I would remove (in ms) from the description and add a new field "*Units:* milliseconds " like we have on cluster properties.